### PR TITLE
WAT-2812: attempt to use AbisBook instead of BookMisc

### PR DESCRIPTION
**Pull request by ebrenner-blurb**

### CODE REVIEW

**Review by atripathi87 — APPROVED** [review]

**Comment by CraigSimmonsBlurb** [comment]
+1 - Some pretty extensive changes here EAB ;).  Based on the info you provided to me about the template, this seems reasonable and I don't see any immediate issues here. Would be good to have WT or someone else review as well

**Review by tcampbell-blurb — APPROVED** [review]
I want to note that we're all a bit in the dark with this issue - and if this is a fix. So I'm approving from my limited perspective and would really appreciate someone from WhileTrue also digging into this.

File: lib/ruby-mws/api/feed.rb [hunk 1]
```diff
@@ -197,7 +197,7 @@ def get_addition_data(xml, entry_hash)
         xml.ProductData {
           xml.Books {
             xml.ProductType {
-              xml.BooksMisc {
+              xml.AbisBook {
```
Comment (boris-v) [inline lib/ruby-mws/api/feed.rb:200]: Have you verified that the AbisBook template doesn't have new "required fields" requirements (vs. BooksMisc) ?
I can't search or get to the template specs because I don't have a MWS account.

**Comment by tcampbell-blurb** [comment]
I've added more notes along with Boris's notes.

File: lib/ruby-mws/api/feed.rb (hunk 1 above)
Comment (ebrenner-blurb) [reply lib/ruby-mws/api/feed.rb:200]: Yes I reviewed their XSD file and:
1) All the fields we add are also applicable to AbisBook.
2) There are no required fields in AbisBook.

**Review by tcampbell-blurb — CHANGES_REQUESTED** [review]

File: lib/ruby-mws/api/feed.rb (hunk 1 above)
Comment (ebrenner-blurb) [reply lib/ruby-mws/api/feed.rb:200]: @tcampbell-blurb this is the xsd schema for book submissions that I found in seller central's help directory https://images-na.ssl-images-amazon.com/images/G/01/rainier/help/xsd/release_1_9/Books.xsd

If you look under the abis books tag, every subelement has minOccurs=0 which I interpret to mean is optional.

The documentation is pretty sparse and hard to follow, I'm hoping that this at least gives us a more useful error message even if it doesn't solve the issues outright.

**Comment by ebrenner-blurb** [comment]
Merging per discussion with Tim.
